nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -202,7 +202,17 @@ protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
   protected String defaultWarehouseLocation(TableIdentifier table) {
     String location;
     if (table.hasNamespace()) {
-      location = SLASH.join(warehouseLocation, table.namespace().toString(), table.name());
+      String baseLocation;
```

Comment:
maybe define the base location here directly? `String baseLocation = SLASH.join(warehouseLocation, table.namespace().toString())`.

Then you could use it directly in as the fallback: `loadNamespaceMetadata(table.namespace()).getOrDefault(NAMESPACE_LOCATION_PROPS, baseLocation)`

Reply:
I more use to code in scala so without reaffecting the variable,
But i change the code. 
Thanks for the review